aiinfra-cluster/main.tf

```diff
@@ -194,3 +201,17 @@ module "aiinfra-default-dashboard" {
   widgets         = var.orchestrator_type != "gke" ? concat(local.nvidia_widgets, local.gce_gke_gpu_utilization_widgets) : local.gce_gke_gpu_utilization_widgets
   depends_on      = [module.dashboard-widget-data]
 }
+
+module "aiinfra-k8s-setup" {
+  source              = "./modules/kubernetes-ops"
+  project             = var.project_id
+  gke_conn            = {
+    gke_cluster_endpoint           = module.aiinfra-compute.gke_cluster_endpoint
+    gke_certificate_authority_data = module.aiinfra-compute.gke_certificate_authority_data
+    gke_token                      = module.aiinfra-compute.gke_token
+  }
+  enable_k8s_setup                     = local.kubernetes_setup_config.enable_k8s_setup
+  kubernetes_service_account_name      = local.kubernetes_setup_config.kubernetes_service_account_name
+  kubernetes_service_account_namespace = local.kubernetes_setup_config.kubernetes_service_account_namespace
+  node_service_account = local.kubernetes_setup_config.node_service_account
+}
```

Comment:
does this need a `count = var.orchestrator_type == "gke" ? 1 : 0` in it or will the `provider "kubernetes"` just not not worry about it?